src/command_modules/azure-cli-documentdb/azure/cli/command_modules/documentdb/custom.py

```diff
@@ -0,0 +1,48 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# --locations
+# -ipRangeFilter
+# -defaultConsistencyLevel
+# - MaxIntervalInSeconds
+# - MaxStalenessPrefix
+# - resource-group-location
+# -Kind
+
+from azure.cli.command_modules.documentdb.sdk.models import (
+    ConsistencyPolicy,
+    DatabaseAccountCreateUpdateParameters,
+    DatabaseAccountKind,
+    DatabaseAccountOfferType,
+    DefaultConsistencyLevel,
+    Location
+)
+
+def cli_documentdb_create(client,
+                          resource_group_name,
+                          name,
+                          resource_group_location,
+                          locations,
+                          default_consistency_level=None,
+                          max_staleness_prefix=100,
+                          max_interval_in_seconds=5,
+                          ip_range_filter=None):
+    # pylint:disable=line-too-long
+    """Create new DocumentDB Database Account
+    :param resource_group_name: Name of resource group
+    :param name: Name of DocumentDB Database Account
+    :param location: Resource group location
+    """
+    consistency_policy = None
+    if not default_consistency_level is None:
+        consistency_policy = ConsistencyPolicy(default_consistency_level, max_staleness_prefix, max_interval_in_seconds)
+
+    params = DatabaseAccountCreateUpdateParameters(
+        resource_group_location,
```

Comment:
Why is resource_group_location required? If you specify the resource_group_name, the backend should be able to determine what the location is. Regardless, since you specify the resource_group_name, you should just look up the resource_group_location instead of making it something the user has to know and set.